stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -598,7 +598,13 @@ internal class ChatDomainImpl internal constructor(
         // 0 ensure load is complete
         initJob.join()
 
-        // 1 update the results for queries that are actively being shown right now
+        // 1 Watch all channels that failed, it is needed for channels that are created locally, in other case it won't be created on the server
+        activeChannelMapImpl
+            .values
+            .filter { it.errorWatching }
+            .forEach { it.watch() }
```

Comment:
can be optimized by only calling watch for channels that we didn't receive after doing the query channels

Reply:
This list is going to be ~0 almost always